Cargo.toml

```diff
@@ -12,11 +12,15 @@ documentation = "https://docs.rs/jtd-derive"
 categories = ["encoding"]
 keywords = ["schema", "codegen", "json"]
 
+[features]
+url = ["dep:url"]
+
```

Comment:
```suggestion
```

I'm fairly sure this is unnecessary. [The feature should be implicitly created bc of `optional = true`.](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies)

Reply:
TIL